master/static/srv/command-entrypoint.sh

```diff
@@ -3,8 +3,19 @@
 source /run/determined/task-signal-handling.sh
 source /run/determined/task-logging-setup.sh
 
+if [ -z "$DET_PYTHON_EXECUTABLE" ]; then
```

Comment:
We can't rely on python being available in the container?

Reply:
Just copied what the other entrypoint.sh scripts were doing.

Reply:
Seems like a mess -- who exactly would `install python3` at this point?

Reply:
Not sure.  Maybe you can't install it on the host and set the DET_PYTHON_EXECUTABLE to the path where you've bind mounted the directory on the container?

Reply:
People can build their own environment files -- so they might install it as python vs python3.   There is no standardization here AFAICT. :-( 


So make sense to be consistent with other entrypoint scripts.

Reply:
the log shipper already relies on python and already runs this logic, so we are fine.